### PR TITLE
Add a Troubleshooting section to the README

### PR DESCRIPTION
**Pull request by michaelpj**
Good to help people help themselves!

I wanted to populate the 'Common issues' section with some more
examples, any ideas?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -619,6 +620,44 @@ Example with `tasty-discover`:
 ```
 This returns an error in HLS if 'tasty-discover' is not in the path: `could not execute: tasty-discover`.
 
+## Troubleshooting
+
+### Common issues
+
+#### Difficulties with Stack and `Paths_` modules
+
+These are known to be somewhat buggy at the moment: https://github.com/haskell/haskell-language-server/issues/478.
```
Comment (googleson78) [inline README.md:629]: Stack 2.5 is officially released now, so this is "kind of" a solved thing? Still good to have it here for a while maybe..

Comment (jneira) [reply README.md:629]: Agree, with a note pointing out stack versions newer than 2.5 fix it

**Review by jneira — APPROVED** [review]
Definitely this section was being needed, thank you for opening it.
@michaelpj if you dont have time to continue working on this, we can merge it as is and improve it later.

**Comment by michaelpj** [comment]
Updated with a note about the stack version. Hopefully other people will add some more common issues :)

**Comment by jneira** [comment]
great, thanks again. Merging!

**Pull request merged**